src/keyring/KeyringController.ts

```diff
@@ -740,6 +751,114 @@ export class KeyringController extends BaseController<
     await this.#keyring.persistAllKeyrings();
     await this.fullUpdate();
   }
+
+  // Ledger Related methods
+
+  private async addLedgerKeyring(): Promise<LedgerKeyring> {
+    return await this.#keyring.addNewKeyring(KeyringTypes.ledger);
+  }
+
+  /**
+   * Retrieve the existing LedgerKeyring or create a new one.
+   *
+   * @returns The stored Ledger Keyring
+   */
+  async getLedgerKeyring(): Promise<LedgerKeyring> {
```

Comment:
It doesn't appear that this method is being tested directly, should it?

Reply:
[296bef1](https://github.com/MetaMask/controllers/pull/862/commits/296bef199122f8f7608f1697faf2b5a847f62cde)